docs/README.md

```diff
@@ -1044,6 +1044,140 @@ docker-compose up
 2. Site will be avaiable on `localhost:3001`
 3. You can update the documentation by modifying the file `docs/README.md`
 
+## OpReturn
+
+OP_RETURN is a [script opcode](https://wiki.bitcoinsv.io/index.php/OP_RETURN) that allows adding extra information to transactions beyond standard inputs and outputs. This can be useful for applications that want to include:
+
+* Different transaction types related to events (e.g., gaming);
+
+* Invoice numbers;
+
+* Nonces to differentiate payments of the same amount;
+
+* Other data.
+
+### General OpReturn structure
+
+    <OP_RETURN opcode> <pushdata for protocol identifier> <protocol identifier> <version> <pushdata for data payload> <data payload> <pushdata for nonce> <nonce>
+
+### General Syntax Rules:
+
+The data following the OP_RETURN is composed by (n) mandatory components:
```

Comment:
Either this sentence is wrong or number 1. is wrong, since `6a` is the OP_RETURN opcode, so is not part of what follows it.

Also mention "hex-encoded" somewhere here.